docs/gathered-data.md

```diff
@@ -59,7 +59,7 @@ API Reference:
 
 	https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 
-* Location in archive: config/storage/<namespace>/<name>.json
+* Location in archive: config/storage/<namespace>/<resource>/<name>.json
```

Comment:
Will the CephCluster gatherer gather any other CRDs than CephCluster? If not, I suggest to make the path specific and use `cephcluster(s)` instead of `<resource>`.

Reply:
Yeah, I can change it. Basically what I mean by that was exactly it, that it will be `cephclusters`, but I think that it is better to let it explicit.